x/mongo/driver/topology/topology_test.go

```diff
@@ -620,16 +620,17 @@ func TestTopologyConstruction(t *testing.T) {
 			pollingRequired bool
 		}{
 			{"normal", "mongodb://localhost:27017", false},
-			{"srv", "mongodb+srv://localhost:27017", true},
+			// {"srv", "mongodb+srv://localhost:27017", true},
```

Comment:
Why is this commented out?

Reply:
This is an [invalid test](https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst#:~:text=It%20is%20an%20error%20to%20specify%20a%20port%20in%20a%20connection%20string%20with%20the%20mongodb%2Bsrv%20protocol%2C%20and%20the%20driver%20MUST%20raise%20a%20parse%20error%20and%20MUST%20NOT%20do%20DNS%20resolution%20or%20contact%20hosts.) that will result in this error:

```
topology_test.go:628: error constructing topology config: error parsing uri: URI with srv must not include a port number
```